### PR TITLE
Buckets hold 150u, don't fit in machinery

### PR DESCRIPTION
**Pull request by 9600bauds**
To make buckets a little more useful, they now hold 150 units, so it can hold more than twice as much mutagen as before.

However, they now don't fit in machinery such as chemmasters or chemdispensers, to prevent them from becoming the beaker meta.

:cl:
 * tweak: Buckets can now hold 150 units, up from 70. However, they won't fit in machinery, such as chemdispensers.

### CODE REVIEW

**Comment by Probe1** [comment]
nooo make a big bucket I wanted an easy excuse to add roomba cleaning bots but need an item that fits the bill to build them from

**Comment by Zth--** [comment]
I like the idea, but don't you think this is way too much copy paste?

File: code/game/machinery/biogenerator.dm [hunk 1]
```diff
@@ -328,9 +328,11 @@
 		return 1
 	else if(istype(O, /obj/item/weapon/reagent_containers/glass))
 		if(beaker)
-			to_chat(user, "<span class='warning'>The biogenerator already occuped.</span>")
+			to_chat(user, "<span class='warning'>The biogenerator is already occuped.</span>")
```
Comment (PJB3005) [inline code/game/machinery/biogenerator.dm]: > occuped

**Review by PJB3005 — APPROVED** [review]

**Comment by 9600bauds** [comment]
The var/beaker is defined in every individual machinery, so copypaste is sadly needed unless someone went and made a subtype.

File: code/game/machinery/biogenerator.dm (hunk 1 above)
Comment (Exxion) [reply code/game/machinery/biogenerator.dm]: PJB actually proofreading?

**Pull request merged**